mfr/providers/osf/provider.py

```diff
@@ -96,6 +99,12 @@ def __init__(self, request, url):
         # }}
 
         name, ext = os.path.splitext(metadata['data']['name'])
+
+        if MAX_FILE_SIZE_TO_RENDER.get(ext) and metadata['data']['size'] > MAX_FILE_SIZE_TO_RENDER.get(ext):
```

Comment:
Instead of calling `MAX_FILE_SIZE_TO_RENDER.get(ext)` 2 or 3 times, go ahead and save it to an intermediate variable.